### ISSUE TITLE
Conjunctival membrane

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9862]
**Preferred term label:**
Conjunctival membrane

**Synonyms**
with bleeding upon removal

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Conjunctivitis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
added as
HP:0034814
Conjunctival membrane
def: Formation of a sheet of tissue by coagulum of exudate over the conjunctiva of the eye that if removed leaves behind a denuded bleeding surface.
